src/universal/modules/meeting/containers/MeetingContainer/MeetingContainer.js

```diff
@@ -247,29 +245,7 @@ class MeetingContainer extends Component {
   };
 
   gotoAgendaItem = (idx) => async () => {
-    const {atmosphere, viewer: {team: {activeFacilitator, agendaItems, facilitatorPhase}}, myTeamMemberId} = this.props;
-    const isFacilitating = activeFacilitator === myTeamMemberId;
-    const facilitatorPhaseInfo = actionMeeting[facilitatorPhase];
-    const agendaPhaseInfo = actionMeeting[AGENDA_ITEMS];
-    const firstIncompleteIdx = agendaItems.findIndex((a) => a.isComplete === false);
-    const nextItemIdx = firstIncompleteIdx + (facilitatorPhase === AGENDA_ITEMS ? 1 : 0);
-    const shouldResort = facilitatorPhaseInfo.index >= agendaPhaseInfo.index && idx > nextItemIdx && firstIncompleteIdx > -1;
-    if (isFacilitating && shouldResort) {
```

Comment:
To be honest, I have no idea what was going on here other than it was resorting the agenda list in odd ways as you clicked through some (but not all) agenda items. Removing this block of code fixed it 🤷‍♂️

Reply:
this handles float precision failure. not critical code, but without it, folks run the risk of not being able to drag & drop something more than ~40 times.

Reply:
Captured here: https://github.com/ParabolInc/action/issues/1700